src/api/handlers/TutorFindManyHandler.ts

```diff
@@ -0,0 +1,23 @@
+import { Handler } from 'aws-lambda';
+import { TutorService } from '../services/TutorService';
+
+class TutorFindManyHandler {
+    private service: TutorService;
+
+    constructor() {
+        this.service = new TutorService();
+    }
+
+    public execute() {
+        const tutor = this.service.findMany();
```

Comment:
@Aristevao **update** the property name from `tutor` to `tutors`, because when we are storing an array at some variable we use a plural noun.